CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+When contributing to this repository, please follow the following rules to keep a standard and allow safe contributions.
+
+## Test coverage
+
+- The project uses [Jest](https://jestjs.io/) as it's Testing Framework
+- All new changes should have relevant test coverage
+
+## Commits
+
+- Commit messages should be created using `npm run commit`.
+
+  This uses [Commitizen](http://commitizen.github.io/cz-cli/) to format the commit messages in a way that is compatible with [SemanticRelease](https://github.com/semantic-release/semantic-release).
+
+  1. Stage files needed for the commit
+  2. Run `npm run commit`.
+  3. Follow the instructions
+
+- Commit messages should be [signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification).
+
+  To allow this, the recommended method is as follows:
+
+  1. Verify you don't already have `gpg` installed
+
+     ```bash
+     which gpg
+     ```
+
+     If you see something like `/usr/local/bin/gpg` it means you already have it and you can skip to step 3
+
+  2. Install `gpg`
+
+     ```bash
+     brew install gpg
+     ```
+
+  3. Generate `GPG key`. See [Generating a new GPG key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-gpg-key) for reference
+
+     ```bash
+     gpg --full-generate-key
+     ```
+
+     The recommended settings used by the code maintainers are:
+
+     ```
+     (1) RSA and RSA: 1
+     RSA keysize: 4096
+     Key is valid for? (0): 0
+     ```
+
+     If you want to associate another email you can follow the steps in this guide: [Associate another email with gpg key](https://docs.github.com/en/github/authenticating-to-github/associating-an-email-with-your-gpg-key)
```

Comment:
Adding a 2nd email to your gpg key isn't super common -- I'm not sure how much of this guide we want to be about GPG -- I think the link to `Generating a new GPG key` and `Tell git about your signing key` with a short summary of the commands to run is maybe enough to keep this focused on this project.